### PR TITLE
Add conda instructions for October release

### PR DESCRIPTION
**Pull request by natke**
As promised, this PR adds the conda instructions for the release next week.

I've set it to draft so that it doesn't accidentally get merged.

TODO:
- [ ] Check install instructions with the new release
- [ ] Check update instructions with the release
- [ ] Add a conda list qsharp step to confirm update?

Change it to ready for review, when you are ready for review :)

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/43fdbceac7f9339a265eb503d693927d03b4a3f6/articles/install-guide/index.md) | 96 | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3c9f96c3-648b-42f5-b9e2-4e52cd2061cf_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/41e373f9c873a872663b59157d88a256a1d289ba/articles/install-guide/index.md) | 96 | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da3c9808-306b-4e53-8e57-f6ff531c1d10_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/41e373f9c873a872663b59157d88a256a1d289ba/articles/install-guide/update.md) | 86 | 72 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3f0ef191-fa31-4f7e-971d-e625f1b2bca9_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/install-guide/index.md [hunk 1]
```diff
@@ -158,7 +148,7 @@ Depending on your chosen development environment, there are different installati
 1. Pre-requisites
 
    - [VS Code](https://code.visualstudio.com/download)
-   - [.NET Core SDK 2.1 or later](https://www.microsoft.com/net/download)
+   - [.NET Core SDK 3 or later](https://www.microsoft.com/net/download)
```
Comment (bromeg) [inline articles/install-guide/index.md]:
```suggestion
   - [.NET Core SDK 3.0 or later](https://www.microsoft.com/net/download)
```

File: articles/install-guide/index.md [hunk 2]
```diff
@@ -192,7 +182,7 @@ Depending on your chosen development environment, there are different installati
 
 1. Pre-requisites
 
-    - [.NET Core SDK 2.1 or later](https://www.microsoft.com/net/download)
+    - [.NET Core SDK 3 or later](https://www.microsoft.com/net/download)
```
Comment (bromeg) [inline articles/install-guide/index.md]:
```suggestion
    - [.NET Core SDK 3.0 or later](https://www.microsoft.com/net/download)
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a8ecff17b8104a289c183631b7e7c68829e80a64/articles/install-guide/index.md) | 96 | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/90ecfa18-857a-404d-b5b2-876afd303171_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a8ecff17b8104a289c183631b7e7c68829e80a64/articles/install-guide/update.md) | 86 | 72 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c935b54-b1be-469c-bb12-30a5565078d3_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/install-guide/index.md [hunk 3]
```diff
@@ -24,20 +24,12 @@ Depending on your chosen development environment, there are different installati
 1. Pre-requisites
 
     - [Python](https://www.python.org/downloads/) 3.6 or later
```
Comment (gillenhaalb) [inline articles/install-guide/index.md]:
```suggestion
    - [Python](https://www.python.org/downloads/) 3.6 or 3.7 (3.8 not yet supported)
```

File: articles/install-guide/index.md [hunk 4]
```diff
@@ -85,15 +77,13 @@ Depending on your chosen development environment, there are different installati
...
```
Comment (gillenhaalb) [inline articles/install-guide/index.md]:
```suggestion
    - [Python](https://www.python.org/downloads/) 3.6 or 3.7 (3.8 not yet supported)
```

File: articles/install-guide/index.md [hunk 5]
```diff
@@ -24,20 +24,12 @@ Depending on your chosen development environment, there are different installati
...
-
-1. Install the `iqsharp` package
-
-    ```bash
-    dotnet tool install -g Microsoft.Quantum.IQSharp
-    dotnet iqsharp install
-    ```
+    - The [Anaconda](https://docs.anaconda.com/anaconda/install/) distribution
 
 1. Install the `qsharp` package
```
Comment (gillenhaalb) [inline articles/install-guide/index.md]:
```suggestion

1. Create a conda virtual environment for using Q# without conflict

	- In the Anaconda Prompt, enter the following and `y` when prompted:

		```Anaconda Prompt
		conda create -n qsharpEnv python=3.7
		```

	- Once the environment is created, activate it with

		```Anaconda Prompt
		conda activate qsharpEnv
		```
		You can find more information on conda environments [here](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html).

```

**Comment by bromeg** [comment]
Closed--conda is postponed

**Pull request closed without merging**